DuckDuckGo/WidgetEducationView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  WidgetEducationView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+extension Font {
+    init(uiFont: UIFont) {
+        self = Font(uiFont as CTFont)
+    }
+}
+
+@available(iOS 14.0, *)
+struct WidgetEducationView: View {
+    
+    var body: some View {
+        ZStack {
+            Color.background
+                .ignoresSafeArea()
+            ScrollView {
+                VStack(alignment: .leading, spacing: Const.Spacing.paragraph) {
+                    NumberedParagraph(number: 1,
+                                      text: Text(UserText.addWidgetSettingsFirstParagraph))
+                    NumberedParagraph(number: 2,
+                                      text: secondParagraphText,
+                                      image: Image.homeScreen)
+                    NumberedParagraph(number: 3,
+                                      text: Text(UserText.addWidgetSettingsThirdParagraph),
+                                      image: Image.widgetExample)
+                }
+                .padding(.horizontal)
+                .padding(.top, Const.Padding.top)
+            }
+        }
+    }
+    
+    private var secondParagraphText: Text {
+        // https://stackoverflow.com/questions/62168292/what-s-the-equivalent-to-string-localizedstringwithformat-for-swiftuis-lo
+        Text("addWidget.settings.secondParagraph.\(Image.add)")
```

Comment:
This seems like magic, but OK! :)

Reply:
I wish it worked and looked different :(